Library/Homebrew/extend/os/mac/keg.rb

```diff
@@ -62,4 +62,19 @@ def codesign_patched_binary(file)
       #{result.stderr}
     EOS
   end
+
+  def dsymutil
+    binary_executable_or_library_files.each do |file|
+      odebug "Extracting symbols #{file}"
+
+      result = system_command("dsymutil", args: [file], print_stderr: false)
+      next if result.success?
+
+      # If it fails again, error out
+      onoe <<~EOS
```

Comment:
`onoe` will only print an `Error`. You probably want [`ofail`](https://rubydoc.brew.sh/Kernel.html#ofail-instance_method) so that `brew` exits with a non-zero exit code. (Or [`odie`](https://rubydoc.brew.sh/Kernel.html#odie-instance_method) if you want to exit immediately, but that's probably not warranted.)

Reply:
I'm wondering if we want to raise an error at all. It's possible that a formula has stripped the executables, or something else has happened which could cause an error. At that point I think we're OK not to have anything other than an error. What do you think?

(This would certainly be my workflow, especially if I was building a number of formulas with debug-symbols). I wouldn't want my build to fail.

Reply:
If you're specifically requested debug symbols: I think it should error.

Reply:
Done in 4eea117e849a68b9364242808050316696b2d9dd